pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -239,14 +239,14 @@ void setBootnodes(final List<String> values) {
   @Option(
       names = {"--p2p-host"},
       paramLabel = MANDATORY_HOST_FORMAT_HELP,
-      description = "Host for P2P peer discovery to listen on (default: ${DEFAULT-VALUE})",
+      description = "Ip address this node advertises to its peers (default: ${DEFAULT-VALUE})",
```

Comment:
fyi @PegaSysEng/pliny - I'm updating the CLI descriptions here and below.  Let me know if you see any issues.